### PR TITLE
[rviz_rendering] update ogre version + fix deprecated function

### PR DESCRIPTION
**Pull request by jeising**
Adopt to the new bugfix version of ogre. It holds the fix made by @wjwwood.

### CODE REVIEW

File: rviz_rendering/src/rviz_rendering/render_window_impl.cpp [hunk 1]
```diff
@@ -175,11 +175,11 @@ createScene(Ogre::SceneManager * ogre_scene_manager)
   // sphereMaterial->setSelfIllumination(0.2f, 0.2f, 0.1f);
 
   sphereMesh->setMaterialName("SphereMaterial");
-  childSceneNode->setPosition(Ogre::Vector3(0.0f, 0.0f, 0.0f));
+  childSceneNode->setPosition(Ogre::Vector3(0.0f, 0.5f, 0.0f));
```
Comment (wjwwood) [inline rviz_rendering/src/rviz_rendering/render_window_impl.cpp:178]: Why this change?

**Review by wjwwood — CHANGES_REQUESTED** [review]
+1 for updating to 1.10.8, but I think we should rebase the patch, not just avoid it. Not everything is upstream yet.

File: rviz_rendering/src/rviz_rendering/render_window_impl.cpp (hunk 1 above)
Comment (jeising) [reply rviz_rendering/src/rviz_rendering/render_window_impl.cpp:178]: It made the sphere sit up on the mesh ... but this can be discarded.

Comment (wjwwood) [reply rviz_rendering/src/rviz_rendering/render_window_impl.cpp:178]: Yeah, I think this code is now obsolete, since we're loading the scene based on the display plugins.

**Comment by jeising** [comment]
Implemented desired changes

**Review by wjwwood — APPROVED** [review]
lgtm, let me try it out on my machine to see if it works.

**Comment by wjwwood** [comment]
The patch isn't applying for me:

```
[ 12%] Performing patch step for 'ogre-1.10.8'
patching file CMake/Templates/OGREConfig.cmake.in
Reversed (or previously applied) patch detected!  Skipping patch.
1 out of 1 hunk ignored -- saving rejects to file CMake/Templates/OGREConfig.cmake.in.rej
patching file CMake/Utils/OgreConfigTargets.cmake
Hunk #1 FAILED at 55.
1 out of 1 hunk FAILED -- saving rejects to file CMake/Utils/OgreConfigTargets.cmake.rej
patching file CMakeLists.txt
Reversed (or previously applied) patch detected!  Skipping patch.
2 out of 2 hunks ignored -- saving rejects to file CMakeLists.txt.rej
make[2]: *** [ogre-1.10.8-prefix/src/ogre-1.10.8-stamp/ogre-1.10.8-patch] Error 1
make[1]: *** [CMakeFiles/ogre-1.10.8.dir/all] Error 2
```

I'll have a look and see if it is just the white space or something.

**Comment by wjwwood** [comment]
Ok, after 5da5d84 and 99cac11 it works for me.
